docs/en/sql-reference/functions/date-time-functions.md

```diff
@@ -1742,14 +1745,14 @@ Result:
 ```
 
 
-## date\_diff
+## date_diff
```

Comment:
`\_` here is required as far as I remember

Reply:
I didn't know. I thought I readded the `\_` due to [a revert](https://github.com/ClickHouse/ClickHouse/pull/63157/files), and I was trying to remove it back. Is there an easy way to check if it's needed or not? Otherwise, I can add it here and move on.

Reply:
I looked at some other examples in the docs and it looks like I was wrong and it is not needed. We can try to merge it and if everything is ok we can also change `## date\_trunc` and other headers.

Reply:
It is not needed, everything is fine.